api/src/main/java/edu/wpi/first/shuffleboard/api/data/DataType.java

```diff
@@ -9,52 +11,70 @@
  *
  * @see DataTypes
  */
-public interface DataType<T> {
+public abstract class DataType<T> {
```

Comment:
What other things? I don't agree that multiple inheritance is a usecase we should worry about for data types